src/pages/lib/auth/mfa/q/platform/[platform].mdx

```diff
@@ -1,15 +1,778 @@
 export const meta = {
-  title: `Multi-factor authentication`,
+  title: `Manage MFA settings`,
   description: `Learn how to enable multi-factor authentication with Amplify.`,
 };
 
-import js0 from "/src/fragments/lib/auth/js/mfa.mdx";
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{js: js0}} />
+The Auth category supports Multi-factor Authentication (MFA) for user sign-in flows. MFA is an extra layer of security used to make sure that users trying to gain access to an account are who they say they are. It requires users to provide additional information to verify their identity. Amplify Auth supports the MFA methods with Time-based-One-Time Passwords (TOTP) as well as text messages (SMS). In this guide we will review how you can set up MFA using TOTP and SMS and the tradeoffs between these methods to help you choose the right set up for your application. We’ll also review how to set up MFA to remember a device and reduce sign-in friction for your users.
 
-import reactnative0 from "/src/fragments/lib/auth/js/mfa.mdx";
+Before you begin you will need:
+- The Amplify libraries installed and configured
+- A phone number provisioned with [SNS/Pinpoint](https://docs.aws.amazon.com/sns/latest/dg/channels-sms-originating-identities.html) if using SMS
```

Comment:
maybe have if using SMS at the beginning - and add an (optional) or something like that?